benchmarks/src/main/scala/scalaz/tests/TestBenchmark.scala

```diff
@@ -0,0 +1,20 @@
+package scalaz
+package tests
+
+import java.util.concurrent.TimeUnit
+import org.openjdk.jmh.annotations._
+import scala.Array
+import scala.Predef.{ wrapString, String }
+import scalaz.Predef._
+
+@State(Scope.Thread)
+@BenchmarkMode(Array(Mode.Throughput))
+@OutputTimeUnit(TimeUnit.SECONDS)
```

Comment:
oh come on, they don't take _seconds_...

Reply:
Fair, though I don't really use the outputted time. Just the profiler. Also, it shows fractional seconds.